src/main/java/seedu/address/logic/commands/ClientNoteAddCommand.java

```diff
@@ -0,0 +1,84 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.client.Client;
+import seedu.address.model.note.Note;
+
+
+
+/**
+ * todo: add javadocs for ClientNoteAddCommand
+ */
```

Comment:
Extra new lines, and I think the javadoc can be added already right

Reply:
done